Sources/Danger/GitHubDSL.swift

```diff
@@ -465,7 +465,7 @@ extension GitHub {
             case creator
             case description
             case dueOn = "due_on"
-            case id // swiftlint:disable:this identifier_name
+            case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 468;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)